source/manual/rebooting-machines.html.md

```diff
@@ -103,6 +105,24 @@ There is a Fabric task to schedule a machine for downtime in Nagios for
 > manual entry on [adding disks](/manual/adding-disks-in-vcloud.html) for
 > more info.
 
+## Rebooting Cache machines in AWS
+
+The `cache` machines run the `router` app which handles live user traffic.
+To safely reboot them without serving too many errors to users, we must
+remove them from the AWS load balancer target group before rebooting:
+
+1. Login to AWS Console. Switch to the prod environment (`gds aws govuk-production-<your-role> -l`)
+1. Find the [Instance ID](https://eu-west-1.console.aws.amazon.com/ec2/home?region=eu-west-1#Instances:sort=desc:launchTime) of the critical machine(s) (probably all 8 `blue-cache` machines)
+1. `ssh [ip].eu-west-1.compute.internal` (find this from the reboots required alert listing)
+1. Remove the machine from the following [Target Groups](https://eu-west-1.console.aws.amazon.com/ec2/home?region=eu-west-1#TargetGroups:sort=targetGroupName):
+   1. cache-assets-origin
+   1. cache-www
+   1. cache-www-origin
+1. Check traffic again (should only be Smokey healthchecks now).
```

Comment:
Does step (1, well there all 1, the 3rd step) involve checking the traffic? (as I don't think it's mentioned prior to this point)

Reply:
These were amended from the Trello card for the router restarts. No, the "again" should go.